test/tpu/cloudbuild.yaml

```diff
@@ -0,0 +1,60 @@
+substitutions:
+  # GCS location to write temporary checkpoints and summaries.
+  _GCS_BUCKET: 'gs://pytorch-xla-tpu-ci-tests/'
+  # Name of GKE cluster in which to run Job.
+  _CLUSTER_NAME: 'tpu-cluster'
+  # Location of GKE cluster.
+  _CLUSTER_ZONE: 'europe-west4-a'
```

Comment:
if a particular zone is stocked out, what happens. do we have a way to try another zone?

Reply:
Stockouts won't be an issue because the TPUs are created in advance. If all of the TPUs in our pool are occupied, the request will have to wait for available resources. It will take some experimentation to get the TPU pool size right.